modules/preview-react/InformationHighlight/README.md

```diff
@@ -0,0 +1,10 @@
+# Canvas Kit Information Highlight
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/preview-react/README.md">
+  <img src="https://img.shields.io/badge/PREVIEW-beta-blueviolet" alt="PREVIEW: Beta" />
+</a>  This component is work in progress and currently in prerelease.
+
+A information highlight component to highlight and prioritize content
+
+View the [documentation for Information Highlight](LINK_HERE)
+on Storybook.
```

Comment:
```suggestion
View the [documentation for Information Highlight](LINK_HERE)
on Storybook.
```

Reply:
This should be the link: https://workday.github.io/canvas-kit/?path=/docs/preview-information-highlight--docs